awc/src/middleware/redirect.rs

```diff
@@ -347,4 +361,61 @@ mod tests {
 
         assert_eq!(res.status().as_u16(), 302);
     }
+
+    #[actix_rt::test]
+    async fn test_redirect_headers() {
+        let srv = actix_test::start(|| {
+            async fn root(req: HttpRequest) -> HttpResponse {
+                if req
+                    .headers()
+                    .get("custom")
+                    .unwrap_or(&HeaderValue::from_str("").unwrap())
+                    == "value"
+                {
```

Comment:
```suggestion
                    .get("custom")
                    == Some("value")
                {
```

Reply:
The output of get is Result<HeaderValue>.
So it can be like: `== Some(&HeaderValue::from_str("value").unwrap())` if you think is better 🙈 
I can have this, but I prefer not, it's not safe enough: `== HeaderValue::from_str("value").ok().as_ref()`

Reply:
If it is `Result`, then you can write  `== Ok("value")` because `HeaderValue: PartialEq<str>`. Anyhow this may be pedantic, you can ignore it. Thanks for the PR.

Reply:
Oh sry I wrote wrong, it's an `Option`.
Any way, for `== Some("value")` shows this error:

```
372 |                     == Some("value")
    |                             ^^^^^^^ expected struct `HeaderValue`, found `str`
```

Reply:
Oh, sorry, then just ignore it.